source/includes/api/_dashboard.md

```diff
@@ -148,23 +150,25 @@ curl -X GET https://api.resourcewatch.org/v1/dashboard?includes=user
 
 When creating a dashboard, the `application` field should be present and cannot contain any values that are not associated with the creating user's account. If an `application` value is not provided, `["rw"]` is used by default, and the process will fail if the user account does not belong to it. Any role can create a dashboard.
 
-Supported fields:
+**The field `is-highlighted` can only be provided on creation by ADMIN users.** If you provide this field and the token does not match an ADMIN user, the request will fail with status code **403 Forbidden** and the message `You need to be an ADMIN to create/update the is_highlighted attribute of the dashboard.`
```

Comment:
add this to the "description" below, and shorten it. if we document permissions for every field this way, it becomes unmanageable

Reply:
Done!